main.go

```diff
@@ -353,13 +353,15 @@ Use the "task" command to get details of a task. For example:
 	var (
 		flagContainerName string
 		flagFormat        string
+		flagFilter        string
 	)
 	containerEnvCommand := app.Command("container-env", "List environment variables for the task's container. Use --format to choose the output format")
 	containerEnvCommand.Arg("cluster", "Name of the cluster").Required().StringVar(&argClusterName)
 	containerEnvCommand.Arg("service", "Name of the service. This can be the full AWS service name, or the short one without the service- prefix and -<cluster> suffix").Required().StringVar(&argServiceName)
 	containerEnvCommand.Flag("container", "Name of the container").StringVar(&flagContainerName)
 	containerEnvCommand.Flag("format", "Format to render the environment variable in. The options are: export, shell, docker, table. Defaults to table").
 		Default("table").EnumVar(&flagFormat, "export", "shell", "docker", "table")
+	containerEnvCommand.Flag("filter", "Case-insensitive comma-separated list of variable names to drop").StringVar(&flagFilter)
```

Comment:
Filter to me usually means _only show_ the ones that match. The `--filter` flag for the `services` command behaves that way. Can we have a different name for this? Like `--drop` or `--skip`?

Reply:
Oh, good point. I'll update that